sqlserver/assets/configuration/spec.yaml

```diff
@@ -713,6 +713,29 @@ files:
         type: number
         example: 1800
         display_default: false
+    - name: schemas_collection 
+      description: |
+        Configure collection of schemas. If `database_autodiscovery` is not enabled, data is collected 
+        only for the database configured with `database` parameter.
+      options:
+        - name: enabled
+          description: |
+            Enable schema collection. Requires `dbm: true`. Defaults to false.
+          value:
+            type: boolean
+            example: true
```

Comment:
I think to actually default to false, the `example` here should be `false`? right?

Reply:
let's align the example, agreed.

"I think to actually default to false, the example here should be false? right? It might not matter since this is a dict object" - I don't see any influence from the example value on the actual value. But I checked only for this example.

Reply:
https://github.com/DataDog/integrations-core/pull/17258/files#r1635138397